app/code/Magento/Customer/Controller/Account/LoginPost.php

```diff
@@ -218,7 +223,7 @@ public function execute()
                     );
                 } finally {
                     if (isset($message)) {
-                        $this->messageManager->addErrorMessage($message);
+                        $this->messageManager->addError($message);
```

Comment:
method `addError` is deprecated. Instead you need to use `addComplexErrorMessage` method.
More info here https://store.fooman.co.nz/blog/displaying-complex-error-messages-in-magento-2.html